task/prefetch-dependencies-oci-ta/0.1/prefetch-dependencies-oci-ta.yaml

```diff
@@ -212,8 +212,27 @@ spec:
           update-ca-trust
         fi
 
-        cachi2 --log-level="$LOG_LEVEL" $config_flag fetch-deps \
-          $dev_pacman_flag \
+        cachi2 --log-level="$LOG_LEVEL" fetch-deps \
+          ACTIVATION_KEY_PATH="/activation-key"
+        if [ -d "$ACTIVATION_KEY_PATH" ]; then
+            echo "Detected activation key, registering with RHSM."
+
+            # todo: make sure orgid key is consistent with buildah task and docs
+            subscription-manager register \
+              --org $(cat "/activation-key/orgid") \
+              --activationkey $(cat "/activation-key/activationkey")
```

Comment:
Is the user running inside the container privileged?

> Error: this command requires root access to execute

Reply:
Default UBI container.

Reply:
the buildah task it running as user 0 already so imo this is a non-issue.

Reply:
Checked the buildah task, there's nothing going on in terms of user setting really (apart from ID mapping), so by default this runs as root. I guess the container itself isn't privileged, so while not ideal, I guess it's acceptable, so consider my initial comment retracted.

However, that makes things easier for cachi2 then and since we're assuming the default root user inside the container, this whole RHSM registration should IMO be baked into cachi2 rpm-dnf backend rather than in the tekton task for proper integration - we could also leverage dbus to communicate with RHSM for better error handling rather than dealing with shell.